genai-perf/tests/test_wrapper.py

```diff
@@ -79,8 +79,6 @@ def test_profile_export_filepath(self, monkeypatch, arg, expected_filepath):
             "profile",
             "-m",
             "test_model",
-            "--service-kind",
```

Comment:
Removing some code, since this is a default value and excluding it makes the code less verbose.